doc/user-stories/expert-partitioner.md

```diff
@@ -417,3 +420,143 @@ The specs are based on current version of expert partitioner for TW. The goal is
+      * file system
+      * mount point
+      * label
+  * shows the actions: 'Edit', 'Resize' and 'Delete'
+    * and action 'Edit' is selected
+      * shows dialog to edit the RAID partition (similar to edit disk partition)
+    * and action 'Resize' is selected
+      * shows a dialog to modify the RADID devices
+    * and action 'Delete' is selected
+      * shows a confirm popup to delete the RAID
+      * deletes the RAID
+* and tab 'Used Devices' is selected
+  * shows a table with the columns: Device, Size, F, Enc and Type
+    * and the table is filled out with info of all devices that belong to the RAID
+
+### *(pending) When 'Add RAID' is selected*
+* shows a wizard with 5 steps
+  * select RAID type and devices
+  * select RAID options
+  * select partition role
+    * same wizard step than in partition creation (see above)
+  * set partition attributes (fs, mount point, etc)
+    * same wizard step than in partition creation (see above)
+    * 'File System Id' is not shown (automatically set to Linux RAID)
+  * set encrypt password (optional)
+    * same wizard step than in partition creation (see above)
+
+#### *(pending) When we are in wizard step to select RAID type and devices*
+* shows the following options
+  * RAID Type
+    * RAID 0 (at least 2 devices)
+    * RAID 1 (at least 2 devices)
+    * RAID 5 (at least 3 devices)
+    * RAID 6 (at least 4 devices)
+    * RAID 10 (at least 2 devices)
+  * RAID Name (opcional)
+  * Available Devices
+    * shows a table with the columns: Device, Size, Enc, Type
+      * and the table is filled out with info of available devices for RAID
+  * Selected Devices
+    * shows a table with the columns: Device, Size, Enc, Type
+    * and the table is filled out with info of devices that belong to the RAID
+* shows the actions: 'Add', 'Add All', 'Remove', 'Remove All'
+  * and action 'Add' is selected
+    * move the selected device from 'Available devices' to 'Selected Devices'
+  * and action 'Add All' is selected
+    * move all devices from 'Available devices' to 'Selected Devices'
+  * and action 'Remove' is selected
+    * move the selected device from 'Selected Devices' to 'Available devices'
+  * and action 'Remove All' is selected
+    * move all devices from 'Selected Devices' to'Available devices'
+* shows the actions: 'Top', 'Up', 'Down', 'Bottom' and 'Classify'
+  * and action 'Top' is selected
+    * reorder RAID devices by moving to top the current selected device from 'Selected Devices'
+  * and action 'Up' is selected
+    * reorder RAID devices by moving up the current selected device from 'Selected Devices'
+  * and action 'Down' is selected
+    * reorder RAID devices by moving down the current selected device from 'Selected Devices'
+  * and action 'Bottom' is selected
+    * reorder RAID devices by moving to bottom the current selected device from 'Selected Devices'
+  * and action 'Classify' is selected
+    * shows a dialog to classify the RAID devices
```

Comment:
I guess we should have a line here saying that if you click next and the number of devices is below the minimum it shows an error pop-up and stays here.